tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
-envlist = py27
+envlist = tests, pylama
 
-[testenv:py27]
-deps =
-    mock
-    mozfile >= 1.1
-    mozprocess >= 0.22
-    pylama
-    pytest
-    wptserve >= 1.4.0
+[testenv:tests]
+usedevelop = true
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps = -rrequirements/tests.txt
```

Comment:
You're right, running `tox -e py27` will always run the coverage too, which is what most users would do. I was just thinking that a user _could_ create a virtual environment and just install the test requirements, and call `pytest` directly. I'm fine with leaving this as is, I think I just like the idea of separation based on needs.